packages/graphql/src/server.ts

```diff
@@ -8,7 +8,9 @@ import { createSchema } from './schema';
 import { createGraphqlFetch } from './utils/executor';
 import { requireEnv } from './utils/requireEnv';
 
-const { FUEL_PROVIDER_BETA5 } = requireEnv(['FUEL_PROVIDER_BETA5']);
+const { FUEL_PROVIDER = 'https://beta-5.fuel.network/graphql' } = requireEnv([
+  'FUEL_PROVIDER',
+]);
```

Comment:
would this be better?
```suggestion
const { FUEL_PROVIDER } = requireEnv([
  'FUEL_PROVIDER', 'https://beta-5.fuel.network/graphql'
]);
```